### ISSUE TITLE
Add options for two types of vibration

### ISSUE THREAD

**Issue by rendel470** [ramack/ActivityDiary#273]
I found this program in F-Droid and I really liked it!
But she has some nuances of use.

ABOUT VIBRATIONS (in the main menu and a five-minute vibration)

Vibration of the main menu
 This vibration occurs every time you enter the main window (which is undoubtedly correct, but each return to the main window is again accompanied by vibration and it is times annoyed)
Please make this option:
 Main window vibration - on/off

Vibration every five minutes
 This vibration is not very annoying, except during video viewing (at such moments, a reminder of vibration, it just wastes energy and brings repair of the vibrating mechanism of a smartphone closer)
 Here would be the option of minute intervals?  (0 - disabled and 5/10/15/20/25/30 - minute intervals of such vibration)

Perhaps this is a trifle, but not everyone needs constant (and without settings) vibration.
 I hope someday to see these options in the program.

**Comment by ramack** [comment]
Thanke a lot for the feedback.

As a quick fix you can change the settings in the generic Android settings. Just swipe the notification partly away and select the "settings-gear". There choose the sound and vibration settings you want.

In the long, I agree that the vibration shall not occur. The reason is because I update the notification which shows how long the current activity is already running. This update shall be made silently without a re-notification.

**Comment by rendel470** [comment]
Thanks for the advice about setting the notification curtain.
 And the vibration settings in the program itself ... They really are not needed.

**Comment by ramack** [comment]
Yes, but I still would like to remove the continuous renotification. The update of the time in the notification shall work silently.
